spikeinterface/curation/remove_redundant.py

```diff
@@ -0,0 +1,127 @@
+import numpy as np
+from typing import Union
+
+from spikeinterface import WaveformExtractor, BaseSorting
+
+from ..postprocessing import get_template_extremum_channel_peak_shift
+from ..postprocessing import align_sorting
+
+
+def remove_redundant_units(sorting_or_waveform_extractor, 
+                           align=True, 
+                           unit_peak_shifts=None,
+                           delta_time=0.4,
+                           agreement_threshold=0.2,
+                           duplicate_threshold=0.8,
+                           remove_strategy='minimum_shift',
```

Comment:
I think default should be max_spikes, because it works both with a waveform extractor and a sorting extractor. Also what if the shift is the same? Can you implement that?

Reply:
But it should be available. Let's implement max_spikes and max_amplitude. What if minimum shift is the same? You pick the first one randomly? Or in that case we take max spikes? I think it's ambiguous!